docs/ContribOperators.md

```diff
@@ -1786,7 +1786,7 @@ This version of the operator has been available since version 1 of the 'com.micr
 ### <a name="com.microsoft.LongformerAttention"></a><a name="com.microsoft.longformerattention">**com.microsoft.LongformerAttention**</a>
 
   Longformer Self Attention with a local context and a global context. Tokens attend locally: Each token
-  attends to its W previous tokens and W succeding tokens with W being the window length. A selected few tokens
+  attends to its W previous tokens and W succeeding tokens with W being the window length. A selected few tokens
```

Comment:
This file is auto generated by tool.

Shall also change the source: https://github.com/microsoft/onnxruntime/blob/0c6037b5abe571fc43a55ef7a9d2f846820fbe5d/onnxruntime/core/graph/contrib_ops/bert_defs.cc#L103